Mathlib/Data/Nat/Digits/Defs.lean

```diff
@@ -34,7 +34,7 @@ variable {n : ℕ}
 /-- (Impl.) An auxiliary definition for `digits`, to help get the desired definitional unfolding. -/
 def digitsAux0 : ℕ → List ℕ
   | 0 => []
-  | n + 1 => [n + 1]
+  | n + 1 => [n+1]
```

Comment:
Just FYI, there are still examples of  `[n+1]` here and below.

Reply:
Thanks!  This PR started in the wrong way: I should not have used `sed` indiscriminately to begin with.  I'll prepare the future PRs with better tools!